docs/src/debug.md

```diff
@@ -109,26 +109,32 @@ composite selectors.
 # Linux/macOS
 PWDEBUG=console npm run test
 
-# Windows
+# Windows with cmd.exe
 set PWDEBUG=console
+# Windows with PowerShell
+$env:PWDEBUG="console"
 npm run test
```

Comment:
Let's copy the commands as well? Basically, a separate section like Linux/macOS above.